motioneye/templates/main.html

```diff
@@ -286,6 +286,11 @@
                             <td class="settings-item-label"><span class="settings-item-label">{{ _("Kamerao Tipo") }}</span></td>
                             <td class="settings-item-value"><input type="text" class="styled device camera-config" id="deviceTypeEntry" readonly="readonly"></td>
                         </tr>
+                        <tr class="settings-item">
+                            <td class="settings-item-label"><span class="settings-item-label">{{ _("Admin only access") }}</span></td>
```

Comment:
I think a dash is common in this case? If not, the extended log messages I suggested above could be adjusted (removing the dash there).
```suggestion
                            <td class="settings-item-label"><span class="settings-item-label">{{ _("Admin-only access") }}</span></td>
```